content/benefits/Education-Benefits/release-notes/2023-06-09

```diff
@@ -0,0 +1,21 @@
+We've added the new Education Benefits API to the developer portal. It replaces the EVSS Education Services API. 
+
+**Important**: To get production access for this API, you must work for VA or have specific VA agreements. If you have any questions, [contact us](https://developer.va.gov/support/contact-us).
+
+The Education Benefits API returns information about a Veteran’s education benefits, including:
+
+- Whether a Veteran is eligible for the Post-9/11 GI Bill’s education benefits.
+- The total education benefits earned by the Veteran.
+- The amount of a Veteran’s education benefits that has been used.
+- The remaining amount of a Veteran’s education benefit.
+- Details of educational enrollments for which benefits were used, including start and end dates, schools, and credits received. 
+
+
+**Authentication and Authorization**
+
+This API uses the client credentials grant (CCG) flow to provide an access token to trusted systems. See our [CCG guide](https://developer.va.gov/explore/authorization/docs/client-credentials?api=claims) for more details. 
```

Comment:
Ideally, this would link to the CCG documentation specifically for the Education Benefits API, which I'm assuming isn't released yet?
```suggestion
This API uses the client credentials grant (CCG) flow to provide an access token to trusted systems. See our [CCG guide](https://developer.va.gov/explore/authorization/docs/client-credentials?api=claims) for more details. 
```